src/vm/runtimehandles.h

```diff
@@ -198,9 +198,7 @@ class RuntimeTypeHandle {
     void QCALLTYPE GetDefaultConstructor(EnregisteredTypeHandle pTypeHandle, QCall::ObjectHandleOnStack retMethod);
 
     static FCDECL1(ReflectClassBaseObject*, GetDeclaringType, ReflectClassBaseObject* pType);
-#ifdef FEATURE_REMOTING	
```

Comment:
It would be better to change this to unconditionally return false on the managed side under `!FEATURE_REMOTING` ifdef.